.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,66 @@
+name: "Bug report"
+description: "Use this template if you're running into issues with Windows, macOS, or Linux (native package) version."
+labels: bug
+body:
+  - type: checkboxes
+    id: unique
+    attributes:
+      label: "This issue is unique."
+      options:
+      - label: "I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug."
+        required: true
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: "If you know this bug occurs on multiple operating systems, select all you have tested."
+      multiple: true
+      options:
+      - Windows XP / Vista
+      - Windows 10 / 8.1 / 7
+      - macOS 10.11 x86
+      - macOS 10.12 or higher
+      - Linux (DEB package)
+      - Linux (RPM package)
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "Version information"
+      description: "Please provide the exact version of the app you are using. If you don't know what version you are using, look at the 'About' tab in the OnlyOffice DesktopEditors. Before submitting an issue, ensure you are using the [latest version of the app](https://www.onlyoffice.com/en/download-desktop.aspx)."
```

Comment:
We officially stylize as ONLYOFFICE, please change it from OnlyOffice

Reply:
I tink in general it can be shorten to

```
Please, provide the exact version of the app you are using. If you don't know it, look at the 'About' tab in ONLYOFFICE Desktop Editors.
```

Reply:
> I will correct these but to be sure - you want me to also delete the part "Before submitting an issue, ensure you are using..." right?

No, not this should be left as it is, it's shorten variant of `If you don't know what version you are using` - replace on `If you don't know it`

> It is DesktopEditors or Desktop Editors?

I think you're right, we got no official docs but my colleagues are noted to me that `Desktop Editors` are two words, so please replace it also )

Thank you you're very attentive to small details )